<script>

    function add() {

        try {

            let result = a + b;

        } catch (err) {

            console.log(err);

        }

        // let result = a+b;

    }

    add();

    console.log("code is running");

</script>